.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+<!--
+This template is for bug reports. If you are here for another reason, please see below:
+
+1. To propose a new rule, see http://eslint.org/docs/developer-guide/contributing/new-rules
+2. To request a change, see http://eslint.org/docs/developer-guide/contributing/changes
+3. If you have any questions, please stop by our chatroom: https://gitter.im/eslint/eslint
```

Comment:
Might be a dumb question, but I assume markdown will not parse HTML comments and so numbering each line `1.` won't work as it usually would?